CppCustomVisualizer2/Cpp.props

```diff
@@ -0,0 +1,41 @@
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Is this just a duplicate of https://github.com/microsoft/ConcordExtensibilitySamples/blob/main/CppCustomVisualizer/Cpp.props?

Can you make a folder at the root of the repo called `build` and in it create a folder called `targets` and drop this file there? Then have both projects reference this file.

Reply:
Sure

Reply:
Looks like hello world also contains a cpp.prop file that has the same content. Should I the common cpp.props for that project as well?

Reply:
yep, I would prefer to have less PropertyItems to update when theres a breaking change in VS.